src/contrib/cluster/Akka.DistributedData.LightningDB/Akka.DistributedData.LightningDB.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageTags>$(AkkaPackageTags);network;cluster;crdt;replication;lightningdb;lmdb</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.6' ">1.6.1</NetStandardImplicitPackageVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
What C# 8 features are we using here? Asking because some of them - default interfaces - create binary compatibility problems for end-users.

Reply:
Also, if this is good to go now - we should remove the `<VersionSuffix>beta</VersionSuffix>` tag from it. Graduates the LigntingDb package to a mature module status.

Reply:
it's used for `using`, so I don't have to have confusing nested brackets in the code, I can always remove them

Reply:
Regarding the beta suffix, I'd rather wait until MNTK came out positive, since the LMDB does not have any other test except for the MNTK.